packages/sidenav/src/components/SideBar/SideBar.styled.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from "prop-types";
+import styled, { css } from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+
+import { FULLSCREEN, PUSH, OVERLAY } from "../../constants/variants";
+
+const fullScreenStyle = css`
+  top: 0;
+  padding: ${({ expanded }) => (expanded ? "16px" : "16px 0")};
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  
+  ${
+    "" /* keep 1px width when collapsed to prevent safari from lose backdrop-filter setting it to 0 */
+  }
+  width: ${({ expanded }) => (expanded ? "100%" : "1px")};
+  transform: translate(${({ expanded }) => (expanded ? "0" : "1px")});
+
+  backdrop-filter: blur(8px);
+  background: ${({ backgroundColor }) => backgroundColor}CC;
```

Comment:
Keep a background without `CC` so it won't be completely transparent in browsers that don't support `#rrggbbaa` :
```suggestion
  background: ${({ backgroundColor }) => backgroundColor};
  background: ${({ backgroundColor }) => backgroundColor}CC;
```